plugins/modules/get_entry.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/python
+#
+# GNU General Public License v3.0+ (see https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: get_entry
+short_description: Get table/entry from SONiC configuration database
+version_added: "0.2.0"
+description: Manage VLANs in SONiC
+options:
+    table:
+        description: The table to lookup
+        required: true
+        type: str
+    key:
+        description: The key to look up in table
+        type: str
+
+author:
+    - Erik Larsson (@whooo)
+'''
+
+EXAMPLES = r'''
+# Get table
+- name: get vlan table
+  community.sonic.get_entry:
+    table: vlan
+
+# Get entry in table
+- name: Remove VLAN
```

Comment:
I take it this name is a copy/paste error? Otherwise I don't know how it makes sense :D

Reply:
Oops, yes, copy/paste error, fixed now together with the correct table name and removing the ipaddress import